lxqt-session/CMakeLists.txt

```diff
@@ -73,7 +68,6 @@ target_link_libraries(lxqt-session
     Qt5::Widgets
     Qt5::DBus
     Qt5::X11Extras
-    ${XCB_LDFLAGS}
     X11
```

Comment:
@palinek Turns out we need. We still need it for the numlock handling. I just updated the PR.